src/oscar/templates/oscar/dashboard/catalogue/product_class_form.html

```diff
@@ -47,80 +47,82 @@ <h3>{% trans "Sections" %}</h3>
 
             <div class="col-md-9">
                 <div class="tab-content">
-                    {% block product_class_details %}
-                        <div class="tab-pane active" id="product_class_details">
-                            <div class="table-header">
-                                <h3>{% trans "Product Class details" %}</h3>
-                            </div>
-                            <div class="well product-class-details">
-
-                                {% comment %}
-                                    If the ProductClass form has field_errors, partials/form_fields.html
-                                    will render an error message.
-                                    This means that there'll be 2 error messages,
-                                    one from the partial and one from the view. Perhaps there should be
-                                    an option allowing hiding of the error message in the template.
-                                    For now we copy paste what we need from the template.
-                                {% endcomment %}
-
-                                {% if form.non_field_errors %}
-                                    {% for error in form.non_field_errors %}
-                                        <div class="alert alert-error control-group error">
-                                            <span class="error-block"><i class="icon-exclamation-sign"></i> {{ error }}</span>
-                                        </div>
-                                    {% endfor %}
-                                {% endif %}
-
-                                {% for field in form %}
-                                    {% include 'dashboard/partials/form_field.html' with field=field %}
-                                {% endfor %}
-                            </div>
-                        </div>
-                    {% endblock %}
+                    {% block tab_content%}
```

Comment:
I meant the space inside these new tags - `{% block tab_content%}` should be `{% block tab_content %}` (same for the `endblock`). 
The indentation change is fine as well.